assets/js/phoenix_live_view/view.js

```diff
@@ -970,9 +970,9 @@ export default class View {
             finish(null)
           }
         },
-        error: (reason) => reject({error: reason}),
+        error: (reason) => reject(new Error(`failed with reason: ${reason}`)),
```

Comment:
Here I intentionally included a prefix to the reason for grep-ability, as otherwise the message would be solely dynamically coming from the server, making it harder to find.